go/libraries/doltcore/schema/schema.go

```diff
@@ -69,6 +69,13 @@ type Schema interface {
 	// SetPkOrdinals specifies a primary key column ordering. See GetPkOrdinals.
 	SetPkOrdinals([]int) error
 
+	// GetPkPrefixLengths returns a slice of pairs (tag, prefix length) containing the prefix lengths
+	// for any columns that use a text/blob index
+	GetPkPrefixLengths() []uint16
+
+	// SetPkPrefixLengths specifies the tags and prefix lengths for the columns that use text/blob indexes
+	SetPkPrefixLengths([]uint16) error
+
```

Comment:
revert these for now, we'll start with prefixes for secondary indexes only, per offline discussion